packages/common-integration-tests/src/webhooks.ts

```diff
@@ -84,10 +84,6 @@ export function runWebhookTests(
           expect(webhookWeAreLookingFor).toBeTruthy();
         } else if (resp instanceof ListWebhooks.Error) {
           throw new Error(`list webhooks request failed: ${resp.message()}`);
-        } else {
-          throw new Error(
-            `unknown error occured when making a 'listWebhooks' request: ${resp.toString()}`
-          );
```

Comment:
what's the motivation to remove these?

Reply:
They were giving me warning saying these paths are redundant and will never be reached.

Reply:
wow, that's cool.